halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/openstack/OpenstackAccount.java

```diff
@@ -0,0 +1,35 @@
+package com.netflix.spinnaker.halyard.config.model.v1.providers.openstack;
+
+import com.netflix.spinnaker.halyard.config.model.v1.node.Account;
+import com.netflix.spinnaker.halyard.config.model.v1.node.LocalFile;
+import com.netflix.spinnaker.halyard.config.model.v1.node.Validator;
+import com.netflix.spinnaker.halyard.config.problem.v1.ConfigProblemSetBuilder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class OpenstackAccount extends Account {
+    private String authUrl;
+    private String username;
+    private String password;
+    private String projectName;
+    private String domainName;
+    private Boolean insecure = false;
+    @LocalFile private String userDataFile;
+    private OpenstackLbaasOptions lbaas = new OpenstackLbaasOptions();
+    private String regions;
+
+    //TODO(emjburns): add support for rosco options
```

Comment:
Up to you, those options will wind up in `rosco.yml` and won't be in this class, so I figured it'd be easiest to point out to avoid future confusion.